PetGame/wwwroot/css/main.css

```diff
@@ -0,0 +1,4 @@
+.body-container {
```

Comment:
This file should not be included, because it is auto-generated from LESS. Remove it.